### PR TITLE
add BEIS residual fuel mix EFs UK 2022

### PR DESCRIPTION
**Pull request by gpantelidou**
**Is this a new addition or an update to an existing factor?**

- [x] New
- [ ] Update

**What category or categories does it affect?**

Electricity

**What?**

added BEIS residual fuel mix EFs UK 2022

**Why?**

we would like to have in the OEFDB residual fuel mix data from UK government

### CODE REVIEW

**Review by maryarbab — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -21784,3 +21784,9 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_residual_mix_coal,Electricity supplied from grid - residual fuel mix (coal),kWh,not-supplied,1.002,1.002,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,BEIS,Electricity Fuel Mix Disclosure,2022,2021,,GB,p,Climatiq,2023/03/03,Emission intensity of residual fuel mix as reported. Published by BEIS and retrieved from the Fuel mix disclosure data table for the period April 2021 to March 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.gov.uk/government/publications/fuel-mix-disclosure-data-table
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21787]: Not needed - delete

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -21784,3 +21784,9 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_residual_mix_natural_gas,Electricity supplied from grid - residual fuel mix (natural gas),kWh,not-supplied,0.372,0.372,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,BEIS,Electricity Fuel Mix Disclosure,2022,2021,,GB,p,Climatiq,2023/03/03,Emission intensity of residual fuel mix as reported. Published by BEIS and retrieved from the Fuel mix disclosure data table for the period April 2021 to March 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.gov.uk/government/publications/fuel-mix-disclosure-data-table
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21788]: Not needed - delete

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -21784,3 +21784,9 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_residual_mix_nuclear,Electricity supplied from grid - residual fuel mix (nuclear),kWh,not-supplied,0,0,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,BEIS,Electricity Fuel Mix Disclosure,2022,2021,,GB,p,Climatiq,2023/03/03,Emission intensity of residual fuel mix as reported. Published by BEIS and retrieved from the Fuel mix disclosure data table for the period April 2021 to March 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.gov.uk/government/publications/fuel-mix-disclosure-data-table
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21789]: Not needed - delete

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -21784,3 +21784,9 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_residual_mix_renewables,Electricity supplied from grid - residual fuel mix (renewables),kWh,not-supplied,0,0,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,BEIS,Electricity Fuel Mix Disclosure,2022,2021,,GB,p,Climatiq,2023/03/03,Emission intensity of residual fuel mix as reported. Published by BEIS and retrieved from the Fuel mix disclosure data table for the period April 2021 to March 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.gov.uk/government/publications/fuel-mix-disclosure-data-table
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21790]: Not needed - delete

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -21784,3 +21784,9 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_residual_mix_other,Electricity supplied from grid - residual fuel mix (other),kWh,not-supplied,0.795,0.795,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,BEIS,Electricity Fuel Mix Disclosure,2022,2021,,GB,p,Climatiq,2023/03/03,Emission intensity of residual fuel mix as reported. Published by BEIS and retrieved from the Fuel mix disclosure data table for the period April 2021 to March 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.gov.uk/government/publications/fuel-mix-disclosure-data-table
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21791]: Not needed - delete

**Review by TimCroker — CHANGES_REQUESTED** [review]

**Review by maryarbab — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -21784,3 +21784,9 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_residual_mix_average,Electricity supplied from grid - residual fuel mix (average),kWh,not-supplied,0.198,0.198,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,BEIS,Electricity Fuel Mix Disclosure,2022,2021,,GB,p,Climatiq,2023/03/03,Emission intensity of residual fuel mix as reported. Published by BEIS and retrieved from the Fuel mix disclosure data table for the period April 2021 to March 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://www.gov.uk/government/publications/fuel-mix-disclosure-data-table
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21792]: This should be 0.425 - the 0.198 is the average grid mix - not residual. My mistake for not making this clear.

**Review by TimCroker — CHANGES_REQUESTED** [review]

**Pull request closed without merging**